pkgs/test/nixpkgs-check-by-name/src/ratchet.rs

```diff
@@ -22,27 +23,41 @@ impl Nixpkgs {
         validation::sequence_(
             // We only loop over the current attributes,
             // we don't need to check ones that were removed
-            to.package_names.into_iter().map(|name| {
-                Package::compare(&name, from.package_map.get(&name), &to.package_map[&name])
+            to.attribute_names.into_iter().map(|name| {
+                Attribute::compare(
+                    &name,
+                    from.attribute_map.get(&name),
+                    &to.attribute_map[&name],
+                )
             }),
         )
     }
 }
 
-/// The ratchet value for a single package in `pkgs/by-name`
-pub struct Package {
+/// The ratchet value for a single attribute
```

Comment:
This name is super general; I mildly prefer "Package" and I'm interested in what motivated the switch.

Reply:
Oh yeah it's actually still just packages here. I switched the `eval.rs` code from packages -> attributes because it now handles non-package attributes, but that doesn't apply here, will changed back!